airlock/business_logic.py

```diff
@@ -269,11 +269,18 @@ def get_manifest_data(self):
         manifest_path = self.abspath("metadata/manifest.json")
         try:
             return json.loads(manifest_path.read_text())
-        except json.JSONDecodeError as exc:
+        except json.JSONDecodeError:
```

Comment:
It looks like we were intending to use `exc` in the exception message but neglected to make it an f-string.